Dockerfile

```diff
@@ -8,13 +8,14 @@ ENV PATH="/root/.pyenv/bin:/root/.pyenv/shims:${PATH}"
 RUN pyenv install 3.8.5
 RUN pyenv global 3.8.5
 RUN pyenv rehash
-RUN pip3 install scons==4.1.0.post1 pre-commit==2.10.1 pylint==2.7.1 Cython==0.29.22
+RUN pip3 install --no-cache-dir scons==4.1.0.post1 pre-commit==2.10.1 pylint==2.7.1 Cython==0.29.22
```

Comment:
you can also chain all the above runs to reduce the layer count

Reply:
right. Let me do it right away